repose-aggregator/components/filters/keystone-v2-filter/src/main/scala/org/openrepose/filters/keystonev2/KeystoneV2Filter.scala

```diff
@@ -604,7 +604,7 @@ class KeystoneV2Filter @Inject()(configurationService: ConfigurationService,
     private var initialized = false
 
     override def configurationUpdated(configurationObject: SystemModel): Unit = {
-      sendTraceHeader = configurationObject.isTracingHeader
+      sendTraceHeader = configurationObject.getTracingHeader == null || configurationObject.getTracingHeader.isEnabled
```

Comment:
I know it's irrational, but just seeing "null" makes me angry. I propose using `Option(getTracingHeader).isEmpty`